lighthouse-cli/bin.ts

```diff
@@ -44,6 +44,13 @@ interface LighthouseError extends Error {
 
 const cliFlags = getFlags();
 
+if (cliFlags.remoteDevice) {
+  cliFlags.port = 9222;
+  cliFlags.disableNetworkThrottling = true;
```

Comment:
I'll defer to @paulirish if we want to disable network throttling by default in this scenario or just the other three